### ISSUE TITLE
PPTX and other formats are badly recognized and downloaded with .zip extenstions

### ISSUE THREAD

**Issue by dusatvoj** [go-gitea/gitea#15748]
- Gitea version (or commit ref): 1.14.0
- Git version: 2.20.1
- Operating system: Debian 10
  
  
  
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
- Log gist:

## Description

Some people in our company have a problem when they upload `pptx` or `docx` or another OfficeOpenXML. It's downloaded with .zip extension bcs it's badly recognized. It's not happening in linux but in windows because it's `zip` mime type _(some shitty windows stuff but it's annoying for them when they download `presentation.pptx.zip`)_. In linux is downloaded as `pptx` but with zip mime type _(extension does not change)_
I can understand why it's happening but I can't understand why mime type is not set by extension.

**Comment by sapk** [comment]
We use [mime.TypeByExtension](https://github.com/go-gitea/gitea/blob/ab77a24f18ae45248bd6c71778fe3bfacacf864f/modules/public/static.go#L69) in Gitea to determine mime type.
It is based on the host configuration: https://golang.org/src/mime/type.go?s=2843:2882#L91

Maybe you can add those mime types to the host list (and restart gitea after )?

On our side we could maybe add some extra not available on most OS distribution ?

**Comment by dusatvoj** [comment]
I have Debian 10 on server and `mime.types` has extension `pptx` defined as below.
```
application/vnd.openxmlformats-officedocument.presentationml.presentation       pptx
```
But `pptx` is downloaded as zip mime type (screenshot below) ... 
![Screenshot](https://user-images.githubusercontent.com/11560794/117586688-010fa280-b11a-11eb-898b-c563956e6330.png)

Tested in latest firefox on linux

**Comment by zeripath** [comment]
This can now be solved with #15133

**Comment by techknowlogick** [comment]
Closing per linked PR
